### PR TITLE
chore: fix docs search option in mobile view

### PR DESCRIPTION
**Pull request by amareshsm**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

- [ ] Documentation update
- [] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
- [ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
- [ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
- [ ] Add autofix to a rule
- [ ] Add a CLI option
- [ ] Add something to the core
- [x] Other, please explain:

#### What changes did you make? (Give an overview)
In new docs, the search option was not working in mobile view so implemented the search option in mobile view.

After changes:

https://user-images.githubusercontent.com/30730124/169887956-f9fcf464-561e-466d-a03f-440990e308cd.mov

### CODE REVIEW

File: docs/src/assets/js/search.js [hunk 1]
```diff
@@ -82,16 +82,21 @@ function displaySearchResults(results) {
...
-    if (query.length > 2) {
-        fetchSearchResults(query)
-            .then(displaySearchResults)
-            .catch(clearSearchResults);
-    } else {
-        clearSearchResults();
-    }
+searchInputs.forEach((searchInput) => {
+    searchInput.addEventListener('keyup', function (event) {
+        const resultsElement = document.querySelector(`#${searchInput.id}-results`);
```
[the remark this answers is not in the document]
Comment (amareshsm) [reply docs/src/assets/js/search.js]: yes, we can move outside `function (event) {}`.

**Comment by mdjermanovic** [comment]
I'm not sure we copied the intended version of the search feature to this repository. The latest version in the development repo differs significantly, so it might be better to wait for @nzakas to verify if this version needs to be fixed or replaced with another one. 

In particular, there was only one search component, and the same search input and other elements were used on both desktop and mobile (on mobile, search was under the "Index" menu, not under the top menu):

https://docs.eslint.org

**Comment by amareshsm** [comment]
I guess the search component needs to be replaced. https://new.eslint.org/blog/ also uses the same search component.

**Comment by mdjermanovic** [comment]
It seems that this will be fixed by https://github.com/eslint/eslint/pull/15944 ?

**Comment by nzakas** [comment]
Yes, I believe it will be fixed. Let’s hold off on this to verify.

**Comment by nzakas** [comment]
I believe this should be fixed now. @amareshsm can you confirm?

**Comment by amareshsm** [comment]
> I believe this should be fixed now. @amareshsm can you confirm?

yes, it is fixed we can close this PR. Closing this PR in favor of https://github.com/eslint/eslint/pull/15944

**Pull request closed without merging**